hosted/eks/p1/sync_importing_test.go

```diff
@@ -31,6 +31,9 @@ var _ = Describe("SyncImport", func() {
 	When("a cluster is imported for sync", func() {
 		var upgradeToVersion string
 		BeforeEach(func() {
+			if helpers.SkipUpgradeTests {
```

Comment:
This file still has a `var cluster *management.Cluster` declaration.

Reply:
I can't see it, can you check latest commit